homeassistant/components/music_assistant/quality_scale.yaml

```diff
@@ -0,0 +1,70 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: exempt
+    comment: Integration is local push
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
`mock_get_server_info` should not patch internals, but rather the library

Reply:
Actually I investigated patching the library directly, but it significantly complicates the mocking infrastructure and breaks multiple tests. The current approach of mocking config_flow.get_server_info is cleaner and more maintainable. This function is part of our integration layer, not a deep internal